core/src/main/java/com/netease/arctic/catalog/BaseIcebergCatalog.java

```diff
@@ -144,12 +145,27 @@ private org.apache.iceberg.catalog.TableIdentifier toIcebergTableIdentifier(Tabl
   }
 
   public class BaseIcebergTable extends BaseUnkeyedTable {
```

Comment:
Iceberg table will be converted to UnkeyedTable. But i'm curious that why convert to KeyedTable when the iceberg has equality-delete-fileds?

Reply:
UnkeyedTable means table without primary key spec and it is only consist of base store shile KeyedTable consist of base store and change store.

We treat all iceberg tables as UnkeyedTable in version 0.4 to reduce impact on existing code and may refactor ArcticTable API in version 0.5，so iceberg table may have its primary key spec according to its identifier row.